Hakawai/Mentions/HKWMentionsPlugin.h

```diff
@@ -142,8 +165,17 @@ typedef NS_ENUM(NSInteger, HKWMentionsPluginState) {
 
 @class HKWMentionsAttribute;
 
-@interface HKWMentionsPlugin : NSObject <HKWDirectControlFlowPluginProtocol>
+/**
+ This is a temprorary protocol that V1 and V2 version of HKWMentionsPlugin will conform to so we can toggle between two versions easily.
+ Once V2 is fully ramped and tested, we will remove V1 version and this protocol.
+ */
+@protocol HKWMentionsPlugin <HKWDirectControlFlowPluginProtocol>
 
+@property (nonatomic, nonnull, strong) NSCharacterSet *controlCharacterSet;
+@property (nonatomic) NSInteger implicitSearchLength;
+@property (nonatomic, readonly) BOOL implicitMentionsEnabled;
+
+@property (nonatomic) BOOL shouldEnableUndoUponUnregistration;
```

Comment:
i see they were originally from `_mentionsPlugin.h` file, is the plan to move them back after we clean up ?

Reply:
yes. added comment